PlexRequests.Api/SonarrApi.cs

```diff
@@ -64,24 +64,28 @@ public SonarrAddSeries AddSeries(int tvdbId, string title, int qualityId, bool s
             };
 
             var options = new SonarrAddSeries();
-            if (episodes)
-            {
-                options.addOptions = new AddOptions
-                {
-                    ignoreEpisodesWithFiles = true,
-                    ignoreEpisodesWithoutFiles = true,
-                    searchForMissingEpisodes = false
-                };
-            }
-            else
-            {
-                options.addOptions = new AddOptions
-                {
-                    ignoreEpisodesWithFiles = false,
-                    searchForMissingEpisodes = true,
-                    ignoreEpisodesWithoutFiles = false
-                };
-            }
+            
+
+            //I'm fairly certain we won't need this logic anymore since we're manually adding the seasons
+            //if (seasons.Length == 0)
+            //{
+            //    options.addOptions = new AddOptions
+            //    {
+            //        ignoreEpisodesWithFiles = true,
+            //        ignoreEpisodesWithoutFiles = true,
+            //        searchForMissingEpisodes = false
+            //    };
+            //}
+            //else
+            //{
+            //    options.addOptions = new AddOptions
+            //    {
+            //        ignoreEpisodesWithFiles = false,
+            //        ignoreEpisodesWithoutFiles = false,
+            //        searchForMissingEpisodes = true
+            //    };
+            //}

```

Comment:
We can probably just yank this code but wanted you to verify. 

I did a lot of back and forth in Postman to see if this would have an impact on the settings in sonarr and with the other changes we're making I don't think it needs to be here.